test/scale-infra/deploy_ks_cp_infra.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+region=""
+aws_key_name=""
+num_masters=""
+num_workers=""
+instance_type=""
+ks_release="0.24.0"
```

Comment:
The latest regular release is 0.25.1

Reply:
For simplicity removed the default value. The user is now required to input a ks release value.